### ISSUE TITLE
Unable to build docs on node 12 (windows) due to incompatible graceful-fs package dependency

### ISSUE THREAD

**Issue by jafin** [Leaflet/Leaflet#7271]
**Steps to reproduce**
- On windows, have node 12.18 installed.
- npm run docs
- receive `ReferenceError: primordial is not defined.` error

**Expected behavior**
Docs should build

**Current behavior**
```
> leaflet@1.7.1 docs d:\Data\_CodeOS\LEaflet-jafin
> node ./build/docs.js

Building Leaflet documentation with Leafdoc ...
fs.js:35
} = primordials;
    ^

ReferenceError: primordials is not defined
    at fs.js:35:5
    at req_ (d:\Data\_CodeOS\LEaflet-jafin\node_modules\natives\index.js:143:24)
    at Object.req [as require] (d:\Data\_CodeOS\LEaflet-jafin\node_modules\natives\index.js:55:10)
    at Object.<anonymous> (d:\Data\_CodeOS\LEaflet-jafin\node_modules\sander\node_modules\graceful-fs\fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
```

**Environment**
- Leaflet version: 1.7.1
- Browser (with version): N/A
- OS/Platform (with version): Windows 10,  Node 12.18

**Additional context**
The version of graceful-fs that is resolved is not compatible with node 12.
See this for reference https://stackoverflow.com/questions/55921442/how-to-fix-referenceerror-primordials-is-not-defined-in-node

One suggestion is to add a resolutions section to package.json and pin the minimum version of graceful-fs.

**Comment by jafin** [comment]
Ok, confused,  I see the CI server is node14.  Is this a personal config issue?  Does anyone else on Windows node 12 build docs ok?

**Comment by jafin** [comment]
fixed by #7263

**Comment by IvanSanchez** [comment]
Superseded by #7263
